pkg/madmin/api-error-response.go

```diff
@@ -70,7 +71,7 @@ func httpRespToErrorResponse(resp *http.Response) error {
 	if err != nil {
 		return ErrorResponse{
 			Code:    resp.Status,
-			Message: "Failed to parse server response.",
+			Message: fmt.Sprintf("Failed to parse server response: %s.", err.Error()),
```

Comment:
```suggestion
			Message: fmt.Sprintf("Failed to parse server response: %s.", err),
```

why do `err.Error()` whereas error interface always has a Stringer() :-)